pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -221,7 +221,21 @@ self: super: {
   pocket-dns = dontCheck super.pocket-dns;
   postgresql-simple = dontCheck super.postgresql-simple;
   squeal-postgresql = dontCheck super.squeal-postgresql;
-  postgrest = dontCheck super.postgrest;
+  postgrest = unmarkBroken (overrideCabal (drv: {
+    # Upstream requires postgresql to perform tests, but tests fail due to
+    # assumed database user that is not added as part of the test.
+    testToolDepends = drv.testToolDepends or [] ++ [
+      pkgs.postgresql
+      pkgs.postgresqlTestHook
+    ];
+  }) (doJailbreak (dontCheck (super.postgrest.overrideAttrs (old: { src =
+    pkgs.fetchFromGitHub {
+      owner = "postgrest";
+      repo = "postgrest";
+      rev = "v10.0.0";
+      sha256 = "sha256-nqWM5XBD1b4NEM8j5ifKd6VIrjFj09c3cz34Ha8bivk=";
+    };
+   })))));
```

Comment:
Could you send this as a change to the `haskell-updates` branch?

A couple things:

- We don't use `unmarkBroken`, but instead remove the program from `pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml` and regenerate packages with `maintainers/scripts/haskell/regenerate-hackage-packages.sh`.
- We generally don't override the source of Haskell packages (but instead use something like `postgrest_10_0_0` if you want the latest version).
- Using `lib.pipe` would probably let you write this a little cleaner.
- You probably want to define `postgrest` as a top-level package, and call `justStaticExecutables` on it (similar to other Haskell packages on the top-level).  And then reference the top-level package from the NixOS service.